java-springboot/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.getconvey.interview</groupId>
+  <artifactId>java-springboot</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>SpringBoot TakeHome</name>
+
+  <properties>
+    <springboot.version>[2.7.0]</springboot.version>
+    <java.version>14</java.version>
```

Comment:
Is this part of the test? 😆 
java 14 isn't the latest and isn't support by brew. P44 seems to be be mostly 8 and 11 but maybe Last Mile is 14? Haha, anyway, embarrassed to admit this took me a while to figure out how to install, but I'm not a brew master so I thought Iw as doing something wrong. And then I was getting invalid target release errors when trying to `mvn spring-boot:run`
Maybe this kind of troubleshooting is part of the interview? In any case I wasn't able to bring up the spring boot skeleton out of the box and verify but need to timebox this and move on to other things.

Reply:
Last Mile is 14. Stuck till we can get dropwizard removed or upgraded. I'll check what java.version I can specify here to try and add some flexibility.

Reply:
I've verified with 11, 14, 17 and 18 and updated the doc.